### PR TITLE
Adding option to control border thickness for area highlights.

### PR DESCRIPTION
**Pull request by ArslanJafri**

### CODE REVIEW

File: sdk/src/main/java/com/eegeo/mapapi/EegeoMap.java [hunk 1]
```diff
@@ -1026,11 +1026,29 @@ public boolean isMapCollapsed() {
 
     /**
      * Sets highlights on a list of indoor map entities to the specified ARGB color.
+     *
+     * @param indoorMapId ID for the indoor map the entities are in.
```
Comment (ghost) [inline sdk/src/main/java/com/eegeo/mapapi/EegeoMap.java:1030]: UUID

File: sdk/src/main/java/com/eegeo/mapapi/EegeoMap.java [hunk 2]
```diff
@@ -1026,11 +1026,29 @@ public boolean isMapCollapsed() {
...
     public void setIndoorEntityHighlights(final String indoorMapId, final Collection<String> indoorEntityIds, final int highlightColorARGB)
+    {
+        setIndoorEntityHighlights(indoorMapId, indoorEntityIds, highlightColorARGB, 0.5f);
+    }
+
+    /**
+     * Sets highlights on a list of indoor map entities to the specified ARGB color and border thickness.
+     * Note: Border thickness only applies to area highlights.
+     *
+     * @param indoorMapId ID for the indoor map the entities are in.
```
Comment (ghost) [inline sdk/src/main/java/com/eegeo/mapapi/EegeoMap.java:1043]: UUID

**Review by ghost — CHANGES_REQUESTED** [review]
Trivial change requested :)

File: sdk/src/main/java/com/eegeo/mapapi/EegeoMap.java (hunk 1 above)
Comment (ArslanJafri) [reply sdk/src/main/java/com/eegeo/mapapi/EegeoMap.java:1030]: It's mentioned as simply id in rest of the doc comments. Should I update them all to UUID for consistency? 
https://www.wrld3d.com/android/latest/docs/api/EegeoMap/

Comment (ghost) [reply sdk/src/main/java/com/eegeo/mapapi/EegeoMap.java:1030]: OK leave it if it's more consistent with elsewhere

**Review by ghost — APPROVED** [review]

**Pull request merged**